### PR TITLE
[ru] Localize `Infrastructure-as-a-service`

### LINKED ISSUE #3120: [ru] Localize `Infrastructure as a Service (IaaS)`
### Description

This Issue is to localize [`Infrastructure as a Service (IaaS)`](https://github.com/cncf/glossary/blob/main/content/en/infrastructure-as-a-service.md) into Russian.

### Checklist before opening your PR (put `x` in the checkboxes) — note that you have to _add this checklist to your PR and check all the boxes for it to be approved_.
- [] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.
    
**ETA: TBD**

Useful links:
[Как внести свой вклад](https://glossary.cncf.io/ru/contribute/)
[Руководство по стилю](https://glossary.cncf.io/ru/style-guide/)

[Slack channel](https://app.slack.com/client/T08PSQ7BQ/C05G46RMQTX)
[Telegram channel](https://t.me/ru_cloudnative)

### PR DESCRIPTION
**Pull request by kirkonru**
### Describe your changes
Localize `Infrastructure-as-a-service`

### Related issue number or link (ex: `resolves #issue-number`)
resolves #3120 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR uses the **dev-ru** branch as a source/target.
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/infrastructure-as-a-service.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Инфраструктура как услуга (IaaS)
+status: Completed
+category: Technology
+tags: ["infrastructure", "", ""]
+---
+
+Инфраструктура как услуга (Infrastructure as a service, IaaS) — это модель [облачных вычислений](/ru/cloud-computing/), при которой 
+[физические](/ru/bare-metal-machine/) или [виртуализированные](/virtualization/) вычилительные ресурсы,
```
Comment (shurup) [inline content/ru/infrastructure-as-a-service.md:9]:
```suggestion
[физические](/ru/bare-metal-machine/) или [виртуализированные](/virtualization/) вычислительные ресурсы,
```

File: content/ru/infrastructure-as-a-service.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+хранилище и сетевой доступ предоставляются по требованию и оплачиваются по факту потребления. 
+Облачные провайдеры владеют и управляют аппаратным и программным обеспечением, 
+а потребители пользуются им в рамках открытых, закрытых или гибридных развертываний.
```
Comment (shurup) [inline content/ru/infrastructure-as-a-service.md:12]:
```suggestion
а потребители пользуются им в публичных, частных или гибридных средах.
```

File: content/ru/infrastructure-as-a-service.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Какую проблему решает
+
+В случае классических локальных систем организации часто сталкиваются с проблемой эффективного использования вычислительных ресурсов.
```
Comment (shurup) [inline content/ru/infrastructure-as-a-service.md:16]:
```suggestion
В классических локальных (on-premises) окружениях организации часто сталкиваются с проблемой эффективного использования вычислительных ресурсов. 
```

File: content/ru/infrastructure-as-a-service.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+а потребители пользуются им в рамках открытых, закрытых или гибридных развертываний.
+
+## Какую проблему решает
+
+В случае классических локальных систем организации часто сталкиваются с проблемой эффективного использования вычислительных ресурсов. 
+Центры обработки данных приходится строить с учетом потенциального пикового спроса, даже если тот бывает лишь в 1 % случаев. 
+Все остальное время эти вычислительные ресурсы простаивают. 
+С другой стороны, если рабочая нагрузка превышает прогнозируемый спрос, 
+возникает острая нехватка вычислительных ресурсов. 
+Подобное отсутствие масштабируемости приводит к увеличению затрат и неэффективному использованию ресурсов.
```
Comment (shurup) [inline content/ru/infrastructure-as-a-service.md:21]:
```suggestion
Подобные проблемы в масштабируемости приводят к увеличению затрат и неэффективному использованию ресурсов.
```

File: content/ru/infrastructure-as-a-service.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+## Как именно решает проблему
+
+Благодаря IaaS организации могут не тратить средства на покупку и содержание вычислительных мощностей для своих приложений. 
+Инфраструктура по требованию позволяет им арендовать вычислительные ресурсы по мере необходимости,  
+экономя на [капитальных расходах](https://ru.wikipedia.org/wiki/%D0%9A%D0%B0%D0%BF%D0%B8%D1%82%D0%B0%D0%BB%D1%8C%D0%BD%D1%8B%D0%B5_%D1%80%D0%B0%D1%81%D1%85%D0%BE%D0%B4%D1%8B), 
+и при этом обеспечивает гибкость, связанную с возможностью масштабироваться вверх или вниз.
```
Comment (shurup) [inline content/ru/infrastructure-as-a-service.md:28]:
```suggestion
и при этом обеспечивает гибкость в масштабировании вверх или вниз.
```

File: content/ru/infrastructure-as-a-service.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+IaaS снижает первоначальные затраты на эксперименты или тестирование нового приложения и 
+предоставляет возможности для быстрого развертывания инфраструктуры. 
+Облачный провайдер — отличный вариант для сред разработки или тестирования,
```
Comment (shurup) [inline content/ru/infrastructure-as-a-service.md:32]:
```suggestion
Облачные провайдеры отлично подходят для создания сред разработки и тестирования, 
```

**Review by shurup — COMMENTED** [review]
Small corrections only.

**Review by tym83 — APPROVED** [review]

**Review by shurup — APPROVED** [review]

**Pull request merged**